assignments/models.py

```diff
@@ -121,3 +121,55 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+class SchoolClass(models.Model):
+    name = models.CharField(max_length=255)
+
+    class Meta:
+        verbose_name = _('class')
+        verbose_name_plural = _('classes')
+
+    def __str__(self):
+        return self.name
+
+
+class School(models.Model):
+    name = models.CharField(max_length=255)
+    assignment = models.ForeignKey(Assignment, related_name='schools')
+    classes = models.ManyToManyField(SchoolClass, related_name='schools')
+
+    class Meta:
+        verbose_name = _('school')
```

Comment:
These verbose names are not really needed here. While it might be nice to explicitly write the verbose name, it should then be done for all models.

Note!: In the case of `SchoolClass` the verbose name is fine since you are actually changing the name from `School class` to `Class`, but in the case of `School` nothing will change with this verbose name.